### PR TITLE
[MOB-5473] Embedded GA Release

### PR DESCRIPTION
**Pull request by davidtruong**
## 🔹 Jira Ticket(s) if any

* [MOB-5473](https://iterable.atlassian.net/browse/MOB-5473)

## ✏️ Description

> GA ready embedded build branch

[MOB-5473]: https://iterable.atlassian.net/browse/MOB-5473?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by evantk91 — APPROVED** [review]
LGTM

File: iterableapi-ui/build.gradle [hunk 1]
```diff
@@ -37,17 +37,21 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
     implementation 'androidx.appcompat:appcompat:1.0.0'
     implementation 'androidx.recyclerview:recyclerview:1.0.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation 'com.google.android.flexbox:flexbox:3.0.0'
+    implementation 'androidx.cardview:cardview:1.0.0'
+    implementation "com.github.bumptech.glide:glide:4.8.0"
```
Comment (Ayyanchira) [inline iterableapi-ui/build.gradle:43]: Glide library can be discarded. Good that for now its in api-ui and not the api.

File: iterableapi-ui/build.gradle [hunk 2]
```diff
@@ -37,17 +37,21 @@ dependencies {
...
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
     androidTestImplementation 'androidx.test:runner:1.2.0'
     androidTestImplementation 'androidx.test:rules:1.2.0'
-
 }
 
 ext {
     libraryName = 'iterableapi-ui'
-    libraryVersion = '3.4.17'
+    libraryVersion = '3.5.0-beta1'
```
Comment (Ayyanchira) [inline iterableapi-ui/build.gradle:54]: Can be updated when releasing the new version

**Review by Ayyanchira — APPROVED** [review]
Looking great.. 🤞

**Pull request merged**